### PR TITLE
DEVTOOLS-87 Update the root raml in force push

### PR DESCRIPTION
**Pull request by fschargorosdky**

### CODE REVIEW

**Comment by juancoen** [comment]
looks good to me

File: lib/strategies/updateFileStrategy.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    logger.info(messages.uploadingFile(changedFile));
+    return apiPlatformService.updateAPIFile(
+        workspace.bizGroup.id,
+        workspace.api.id,
+        workspace.apiVersion.id,
+        changedFileObject)
+      .tap(function (changedFileResult) {
+        var olderAudit =
+          _.find(workspace.files, 'path', changedFileResult.path).audit;
+        if (olderAudit) {
```
Comment (juancoen) [inline lib/strategies/updateFileStrategy.js]: add a test to check what happens is olderAudit is not truthy

Comment (fschargorosdky) [inline lib/strategies/updateFileStrategy.js]: Done

**Comment by juancoen** [comment]
looks good

**Pull request merged**